artview/components/navigator.py

```diff
@@ -324,12 +324,12 @@ def _openfile(self, filename=None):
             common.ShowWarning(msg)
         return
 
-    def NewFilelist(self, variable, value, strong):
+    def NewFilelist(self, variable, strong):
         '''respond to change in filelist.'''
         if strong:
             self._update_tools()
 
-    def NewFile(self, variable, value, strong):
+    def NewFile(self, variable, strong):
         '''Respond to change in a container (radar or grid).'''
         if hasattr(variable.value, 'filename'):
             # Update the info label.'''
```

Comment:
This seems to have broken the Navigator (and DirectoryList as well) for some reason. It produces the following error:
```
TypeError: NewFile() takes exactly 4 arguments (3 given)
```
From this code it looks like you've just removed the value and updated. Not sure why it thinks it needs an additional argument...